### DISCUSSION TITLE
Add the ability to zoom in on images

### DISCUSSION THREAD

**Discussion by ghost** [Kareadita/Kavita#3356] [Ideas]
### Idea Description

Add the ability to zoom in on images, using for example anime4k, and enlarge the image on the server before transmitting it.

### Idea Category

Feature Enhancement

### Duration of Using Kavita

4 months

### Before submitting

- [x] I've already searched for existing ideas before posting.

**Comment by majora2007** [comment]
I'm assuming this means allow zooming on the PC browser correct? As you can zoom already on mobile.

**Reply by ghost** [reply]
I hope to use tools like anime4k to enlarge the image, so that the resolution of the image can be increased and it will look clearer.
https://github.com/bloc97/Anime4K

**Reply by scare376** [reply]
You're asking for upscaling.

**Reply by scare376** [reply]
It's unlikely to be implemented within Kavita itself but you may want to look at kurp. https://github.com/Snd-R/kurp
